std/experimental/xml/cursor.d

```diff
@@ -0,0 +1,989 @@
+/*
+*             Copyright Lodovico Giaretta 2016 - .
+*  Distributed under the Boost Software License, Version 1.0.
+*      (See accompanying file LICENSE_1_0.txt or copy at
+*            http://www.boost.org/LICENSE_1_0.txt)
+*/
```

Comment:
This comment is redundant with the one below and it's using a different comment style. I think one block of comments is enough.